packages/core/src/account.ts

```diff
@@ -27,6 +27,12 @@ export class AccountSecrets extends Serializable {
     favorites = new Set<VaultItemID>();
 }
 
+/** Various application settings */
+export class AccountSettings extends Serializable {
+    /** Enable emails for failed login attempts */
+    failedLoginAttemptNotifications = true;
```

Comment:
I wonder if the `securityReport` `app.settings` should actually be here, now. What do you think @MaKleSoft ?

Reply:
Yeah, I was actually going to do exactly that when implementing that setting but then decided that it wasn't worth including a new property on `Account` just for that purpose. I think there are a few other settings that should probably persist across devices, like the favicon setting and the "reveal masked fields on hover" setting.

Reply:
Well, the favicon and masking might make sense on the extension and not the desktop/mobile, or vice-versa, so I'd keep those app-specific. I'll move the security report ones here, though.